src/cairoUtilFuncGen/utils/uNselector.ts

```diff
@@ -0,0 +1,409 @@
+    case 'u16':
+      return WM_READ16;
+    case 'u24':
+      return WM_READ24;
+    case 'u32':
+      return WM_READ32;
+    case 'u40':
+      return WM_READ40;
+    case 'u48':
+      return WM_READ48;
+    case 'u56':
+      return WM_READ56;
+    case 'u64':
+      return WM_READ64;
+    case 'u72':
+      return WM_READ72;
+    case 'u80':
+      return WM_READ80;
+    case 'u88':
+      return WM_READ88;
+    case 'u96':
+      return WM_READ96;
+    case 'u104':
+      return WM_READ104;
+    case 'u112':
+      return WM_READ112;
+    case 'u120':
+      return WM_READ120;
+    case 'u128':
+      return WM_READ128;
+    case 'u136':
+      return WM_READ136;
+    case 'u144':
+      return WM_READ144;
+    case 'u152':
+      return WM_READ152;
+    case 'u160':
+      return WM_READ160;
+    case 'u168':
+      return WM_READ168;
+    case 'u176':
+      return WM_READ176;
+    case 'u184':
+      return WM_READ184;
+    case 'u192':
+      return WM_READ192;
+    case 'u200':
+      return WM_READ200;
+    case 'u208':
+      return WM_READ208;
+    case 'u216':
+      return WM_READ216;
+    case 'u224':
+      return WM_READ224;
+    case 'u232':
+      return WM_READ232;
+    case 'u240':
+      return WM_READ240;
+    case 'u248':
+      return WM_READ248;
+    case 'u256':
+      return WM_READ256;
+    default:
+      throw new Error('Invalid CairoUint type');
+  }
+};
+
+export const wmWriteuNImport = (uNtype: CairoUint) => {
+  switch (uNtype.toString()) {
+    case 'u8':
+      return WM_WRITE8;
+    case 'u16':
+      return WM_WRITE16;
+    case 'u24':
+      return WM_WRITE24;
+    case 'u32':
+      return WM_WRITE32;
+    case 'u40':
+      return WM_WRITE40;
+    case 'u48':
+      return WM_WRITE48;
+    case 'u56':
+      return WM_WRITE56;
+    case 'u64':
+      return WM_WRITE64;
+    case 'u72':
+      return WM_WRITE72;
+    case 'u80':
+      return WM_WRITE80;
+    case 'u88':
+      return WM_WRITE88;
+    case 'u96':
+      return WM_WRITE96;
+    case 'u104':
+      return WM_WRITE104;
+    case 'u112':
+      return WM_WRITE112;
+    case 'u120':
+      return WM_WRITE120;
+    case 'u128':
+      return WM_WRITE128;
+    case 'u136':
+      return WM_WRITE136;
+    case 'u144':
+      return WM_WRITE144;
+    case 'u152':
+      return WM_WRITE152;
+    case 'u160':
+      return WM_WRITE160;
+    case 'u168':
+      return WM_WRITE168;
+    case 'u176':
+      return WM_WRITE176;
+    case 'u184':
+      return WM_WRITE184;
+    case 'u192':
+      return WM_WRITE192;
+    case 'u200':
+      return WM_WRITE200;
+    case 'u208':
+      return WM_WRITE208;
+    case 'u216':
+      return WM_WRITE216;
+    case 'u224':
+      return WM_WRITE224;
+    case 'u232':
+      return WM_WRITE232;
+    case 'u240':
+      return WM_WRITE240;
+    case 'u248':
+      return WM_WRITE248;
+    case 'u256':
+      return WM_WRITE256;
+    default:
+      throw new Error('Invalid CairoUint type');
+  }
+};
```

Comment:
We could do a function:
```
(bits) => [[...WARPLIB_MEMORY], `wm_read_${bits}`]
```
Would be less explicit but it wouldn't require us to write us so much code. I think it's a better option, WYT?

Reply:
done, thank you!